### PR TITLE
feat: add user location and bio details to profile page

### LINKED ISSUE #796: Feature: implement user location and additional infos on user profile
### Type of feature

🍕 Feature

### Current behavior

image

### Suggested solution

Include additional info to verified user profile screen

image

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by OgDev-01**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR is still a work in progress...

## Related Tickets & Documents
Fixes #796 

## Mobile & Desktop Screenshots/Recordings
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
Let's go ahead and make this Time Zone here. i.e. UTC+1

Screen Shot 2023-01-25 at 9 49 37 PM

**Review by bdougie — CHANGES_REQUESTED** [review]
This looks fine in regard to the UI, but we should ship the settings page first. Is there anything blocking us from shipping that @OgDev-01?

https://github.com/open-sauced/insights/pull/788

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    className="w-max hover:text-orange-500 "
+                  >
+                    {`@${twitterUsername}`}
+                  </a>
+                </span>
+              )}
+              <span className="flex gap-2 items-center">
+                <HiOutlineMail className="text-light-slate-9" />
+                <Link className="w-max hover:text-orange-500 " href="#">
+                  Send a collaboration request
```
Comment (bdougie) [inline components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:81]: Can we comment this out? I don't feel comfortable shipping a link to this feature until its ready.

**Review by bdougie — CHANGES_REQUESTED** [review]

File: components/organisms/ContributorProfilePage/contributor-profile-page.tsx [hunk 2]
```diff
@@ -73,21 +75,25 @@ const ContributorProfilePage = ({
   const prsMergedPercentage = getPercent(prTotal, prMerged || 0);
   const isLoaded = !loading && !error;
 
+  const { bio, location, name, twitter_username } = user || {};
```
Comment (0-vortex) [inline components/organisms/ContributorProfilePage/contributor-profile-page.tsx:78]: if you put a different error container before this line, you will not need to `|| {};` at the end, as `user` would always be inferred when loaded.

File: components/organisms/ContributorProfilePage/contributor-profile-page.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```
Comment (0-vortex) [inline components/organisms/ContributorProfilePage/contributor-profile-page.tsx:1]: IMHO remove global eslint disables and at worst, only ignore the line in question

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Data fetching patterns are off, otherwise looking good!

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                </Link>
+              </span> */}
+            </div>
+          </div>
+          {interests && (
+            <div className="flex flex-col gap-4 border-b pb-6">
+              <Title className="!text-base !text-light-slate-12" level={5}>
+                Current Interests
+              </Title>
+              <LanguagePill topic="javascript" />
```
Comment (bdougie) [inline components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:85]: We should not hard code this with JavaScript, I'd rather us hide this for now.

**Review by bdougie — CHANGES_REQUESTED** [review]

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx (hunk 4 above)
Comment (OgDev-01) [reply components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:85]: I think the API has been updated with this field... I'll see if i can populate from the Api right away ✌️

**Review by bdougie — APPROVED** [review]
Thanks for the work on this.

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**